css/estilos-index.css

```diff
@@ -0,0 +1,196 @@
+@import url(flexboxgrid.min.css);
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+}
+#enlace-logo{
+    width: 0;
+    float: left;
+    padding: 0;
+    margin: 0 0 0 0;
+}
+#logo-cine{
+    width: 120px;
+    float: center;

```

Comment:
The float property can have one of the following values: right, left, inherit, none. Reference: [9.5.1. Positioning the float: the float property](https://drafts.csswg.org/css2/#propdef-float)